### PR TITLE
feat(extension): add `colorscheme` extension on buildcontext

### PR DESCRIPTION
**Pull request by baimamboukar**
## Basic Checklist

- [ ] ✨ Ran `dartdoc`
- [ ] 🛠️ Added proper documentation
- [x] ❌ No warnings or errors in the code
- [x] ✅ If added a new extension, added in Readme and example too
- [ ] 🧪 If added a new extension, written proper test cases for it

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
         "editor.formatOnSave": true,
         "editor.formatOnType": true,
         "editor.codeActionsOnSave": {
-            "source.organizeImports": true,
-            "source.fixAll": true,
+            "source.organizeImports": "explicit",
```
Comment (AbhishekDoshi26) [inline .vscode/settings.json:6]: Please revert this

**Review by AbhishekDoshi26 — CHANGES_REQUESTED** [review]
See comment.

Also, please add test for the same and add example too.

Also check the codecov pipeline which is failing.

Apart from that, in my opinion, instead of having a colorScheme extension, won't it be beneficial to have Theme extension so that we can access `context.theme`. The reason is, this way, we won't need to create extensions for individual methods/getters of Theme class.

WDYT?

**Comment by baimamboukar** [comment]
Hey @AbhishekDoshi26 👋

Thanks for your prompt review. I took in consideration your observations and refined things you pointed out. please check here https://github.com/AbhishekDoshi26/super_extensions/pull/34

**Comment by baimamboukar** [comment]
Closing this 🥲

**Pull request closed without merging**